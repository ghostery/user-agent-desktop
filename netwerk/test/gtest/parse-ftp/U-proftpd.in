ftp> open ftp.netmanage.com
Connected to ftp.netmanage.com.
220 ProFTPD 1.2.4 Server (Netmanage FTP Server) [156.27.8.3]
Name (ftp.netmanage.com:cyp): 
331 Anonymous login ok, send your complete email address as your password.
230 Anonymous access granted, restrictions apply.
Remote system type is UNIX.
Using binary mode to transfer files.
ftp> ls
200 PORT command successful.
150 Opening ASCII mode data connection for file list.
drwxr-xr-x  13 ftp      ftp           344 Oct 17  2001 pub
drwxrwxr-x   7 ftp      support       304 May 14 16:56 support
-rw-r--r--   1 ftp      ftp           508 Aug 19  1998 welcome.msg
226-Transfer complete.
226 Quotas off
ftp> syst
215 UNIX Type: L8
ftp> close
221 Goodbye.
ftp> 