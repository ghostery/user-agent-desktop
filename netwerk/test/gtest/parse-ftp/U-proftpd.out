<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

10-17-2001  00:00:00       <DIR>          pub
05-14-2002  16:56:00       <DIR>          support
08-19-1998  00:00:00                  508 welcome.msg
